man/magnitude.Rd

```diff
@@ -0,0 +1,60 @@
+\name{magnitude}
+\alias{magnitude}
+\alias{magnitude.breakpointsfull}
+\title{Magnitudes of Breakpoints}
+\description{
+  Computes magnitude statistics for breakpoints.
+}
+\usage{
+\method{magnitude}{breakpointsfull}(object,
+  interval = 0.1, breaks = NULL, component = "trend", \dots)
+}
+\arguments{
+  \item{object}{an object of class \code{"breakpointsfull"} as computed by
+    \code{\link{breakpoints}}.}
+  \item{interval}{the (one-side) interval over which to compute the magnitude for the columns
+    \code{RMSD} and \code{MAD}. If below 1, fraction of the time series length,
+    otherwise number of observations.}
+  \item{breaks}{how many breaks to use or which statistic to use for estimating it,
+    see \code{\link{breakpoints}} for details. If \code{NULL}, uses the defaults.}
+  \item{component}{which covariate(s) used for fitting the \code{breakpointsfull}
+    object to use for magnitude estimation. The components are additive.}
+  \item{\dots}{\emph{ignored}}
+}
+
+\details{
+  The breakpoint magnitude is estimated using several statistics:
+  the difference (\emph{diff}) in the fitted value immediately \emph{before}
+  and \emph{after} the break, and the root mean squared deviance (\emph{RMSD}),
+  mean absolute deviance (\emph{MAD}) and mean deviance (\emph{MD}) between the
```

Comment:
My wikipedia research says [deviation](https://en.wikipedia.org/wiki/Deviation_(statistics)) is [the right word](https://en.wikipedia.org/wiki/Root-mean-square_deviation) in this context, although [deviance](https://en.wikipedia.org/wiki/Deviance_(statistics)) is a valid word as well.

```suggestion
  and \emph{after} the break, and the root mean squared deviation (\emph{RMSD}),
  mean absolute deviation (\emph{MAD}) and mean deviation (\emph{MD}) between the
```